### PR TITLE
this commit completes ticket 4

### PR DESCRIPTION
**Pull request by MonieW**

### CODE REVIEW

File: db/app.js [hunk 1]
```diff
@@ -1,11 +1,35 @@
...
+        response.status(400).send({msg: 'invalid input'})
+    } else next(error)
+    
+})
+
+app.use((error, request, response, next) => {
+    console.log(error);
+    response.status(500).send({msg: 'Internal Server Error'})
+})
+
```
Comment (RichardNeat) [inline db/app.js:31]: You may want to extract this error handlers into their own file at some point to keep things tidy :)

File: db/models/news.models.js [hunk 2]
```diff
@@ -14,5 +14,17 @@ exports.fetchEndpoints = () => {
...
     })
+}
+exports.fetchArticlesById = (article_id) => {
+    return db.query(`SELECT * FROM articles WHERE article_id = $1;`, [article_id])
+    .then(({rows})=> {
+        if(rows.length === 0) {
+            return Promise.reject({status: 404, msg: `id not found`
+        })
+    }
+    console.log(rows[0])
```
Comment (RichardNeat) [inline db/models/news.models.js]: please remove any console logs :)

**Review by RichardNeat — APPROVED** [review]
Nice one Simone! Really good stuff. A couple of suggestions below, up to you if you want to do them now or later but please remove the console log before merging :)

File: db/app.js (hunk 1 above)
Comment (MonieW) [reply db/app.js:31]: Yup, was totally going to do that going forward as they were starting to get more complex, will do on the next ticket

File: db/models/news.models.js (hunk 2 above)
Comment (MonieW) [reply db/models/news.models.js]: those pesky console logs!

**Pull request merged**